scripts/js/lib/api/releaseNotes.ts

```diff
@@ -153,23 +156,36 @@ function addNewReleaseNotes(pkg: Pkg): void {
   pkg.releaseNotesConfig.separatePagesVersions.unshift(pkg.versionWithoutPatch);
 }
 
-function generateReleaseNotesIndex(pkg: Pkg): string {
-  let markdown = `---
-title: ${pkg.title} release notes
-description: New features, bug fixes, and other changes in previous versions of ${pkg.title}.
----
+// function generateReleaseNotesIndex(pkg: Pkg): string {
```

Comment:
There shouldn't ever be commented out code. But taking a step back, we still need this part of the markdown to be generated